instances/atlas/parcellationEntity/JBA/JBA_Area-TE-1-0.jsonld

```diff
@@ -2,9 +2,13 @@
     "@context": {
         "@vocab": "https://openminds.ebrains.eu/vocab/"
     },
-    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-TE-1.0",
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Area-TE-1-0",
     "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_Heschl-s-gyrus"
```

Comment:
I would delete the "-" between Heschl and s

Reply:
so a general rule is to remove all `'`s ?